verify-totp-sms/functions/send-otp.js

```diff
@@ -0,0 +1,37 @@
+exports.handler = async function (context, event, callback) {
+  const response = new Twilio.Response();
+  response.appendHeader('Content-Type', 'application/json');
+
+  try {
+    if (typeof event.to === 'undefined' || event.to.trim() === '') {
```

Comment:
Is there a reason why you didn't use your `detectMissingParams` function here?

Reply:
can't remember if there was a good reason but since there's only one parameter to check it simplifies the code & tests by not having imports. lmk if it's better to change it for consistency.